Trading/Exchange/bybit.py

```diff
@@ -77,48 +82,66 @@ def set_sandbox_mode(self, is_sandboxed):
+                position_dict[ExchangeConstantsOrderColumns.ID.value] = position_dict[
                     ExchangeConstantsOrderColumns.SYMBOL.value]
-            if ExchangeConstantsPositionColumns.ENTRY_PRICE.value not in position \
-                    and self.BYBIT_ENTRY_PRICE in position:
-                position[ExchangeConstantsPositionColumns.ENTRY_PRICE.value] = position[self.BYBIT_ENTRY_PRICE]
-            if ExchangeConstantsPositionColumns.QUANTITY.value not in position \
-                    and self.BYBIT_QUANTITY in position:
-                position[ExchangeConstantsPositionColumns.QUANTITY.value] = position[self.BYBIT_QUANTITY]
-            if ExchangeConstantsPositionColumns.LIQUIDATION_PRICE.value not in position \
-                    and self.BYBIT_LIQUIDATION_PRICE in position:
-                position[ExchangeConstantsPositionColumns.LIQUIDATION_PRICE.value] = position[
+            if ExchangeConstantsPositionColumns.ENTRY_PRICE.value not in position_dict \
+                    and self.BYBIT_ENTRY_PRICE in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.ENTRY_PRICE.value] = position_dict[
+                    self.BYBIT_ENTRY_PRICE]
+            if ExchangeConstantsPositionColumns.QUANTITY.value not in position_dict \
+                    and self.BYBIT_QUANTITY in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.QUANTITY.value] = position_dict[self.BYBIT_QUANTITY]
+            if ExchangeConstantsPositionColumns.LIQUIDATION_PRICE.value not in position_dict \
+                    and self.BYBIT_LIQUIDATION_PRICE in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.LIQUIDATION_PRICE.value] = position_dict[
                     self.BYBIT_LIQUIDATION_PRICE]
-            if ExchangeConstantsPositionColumns.TIMESTAMP.value not in position \
-                    and self.BYBIT_TIMESTAMP in position:
-                position[ExchangeConstantsPositionColumns.TIMESTAMP.value] = position[self.BYBIT_TIMESTAMP]
-            if ExchangeConstantsPositionColumns.MARK_PRICE.value not in position:
-                position[ExchangeConstantsPositionColumns.MARK_PRICE.value] = None  # TODO
-            if ExchangeConstantsPositionColumns.UNREALISED_PNL.value not in position \
-                    and self.BYBIT_UNREALISED_PNL in position:
-                position[ExchangeConstantsPositionColumns.UNREALISED_PNL.value] = position[self.BYBIT_UNREALISED_PNL]
+            if ExchangeConstantsPositionColumns.TIMESTAMP.value not in position_dict \
+                    and self.BYBIT_TIMESTAMP in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.TIMESTAMP.value] = position_dict[self.BYBIT_TIMESTAMP]
+            if ExchangeConstantsPositionColumns.MARK_PRICE.value not in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.MARK_PRICE.value] = None  # TODO
```

Comment:
is it a problem ?

Reply:
Yes we need this data. We need to link this position `market_price` with the data fetched by `MarkPriceUpdater`. I don't know how to do it properly, that's the reason of the `TODO`.